.travis.yml

```diff
@@ -6,10 +6,32 @@ python:
 # http://docs.travis-ci.com/user/workers/container-based-infrastructure/
 sudo: false
 
+addons:
+  sauce_connect: true
+
+env:
+  global:
+    - FEC_WEB_TEST: true
+    - FEC_SELENIUM_DRIVER: remote
+    - secure: SQfanR6GI/1i4v7XPEKT26mKjd1BT/Qt1/5fjrhEIiA4PL2ZWnvArBmcKeq3q3CcVy2KQDTUr4vvPKbo4HGAw1EcUvnJ5lF6v3h0FmZ/bhMdkHk1e/UvfIVjOm0mEO80FDiXRcWEiYgY/Z4LwJ1l2iNC45j/1Odacj9I3fOdteY=
+    - secure: aXkcjh6AHi1MnY1vZ5Nu8/8OexUZ4n8ul5hrM9iex0QdJAm4AoxF0idHybinTAAL7/gSEineX1to6GC3GVnAV/f/FU1mo8sjt0hvwTAx8b+ONR0M0sJW2KrPJRABQK4Dtxu+PQkE4Qim0GaJQ6LBJwTAB+HiegYymqrUFSV5tm0=
+  matrix:
+    - FEC_SAUCE_BROWSER: "false"
+    - FEC_SAUCE_BROWSER: "chrome"
+    - FEC_SAUCE_BROWSER: "firefox"
+
 before_script:
-    - cd openfecwebapp
-install:
-    - pip install -r requirements.txt
+  # Install web app and run in background
+  - pip install -r requirements.txt
+  - npm install
+  - npm install -g browserify
+  - npm run build
+  - python __init__.py &
+  - if [[ $TRAVIS_BRANCH = 'master' ]]; then export FEC_WEB_API_URL="https://api.open.fec.gov"; fi
```

Comment:
Not going to hold up the merge for this, but it might make sense to run off of staging instead of production in this case.